docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -174,6 +177,29 @@ For more details on getting started with the Synthetics Recorder, see <<syntheti
 
 image::images/synthetics-create-test-script-recorder.png[Elastic Synthetics Recorder after recording a journey and clicking Export]
 
+[discrete]
+[[synthetics-request-param]]
+== Make API requests
+
+You can use the `request` parameter to make API requests independently of the browser interactions.
+For example, you could retrieve a token from an HTTP endpoint and use it in a subsequent webpage request.
+
+[source,js]
+----
+step('make an API request', async () => {
+  const response = await request.get(params.url);
+  // Do something with the response
+})
+----
+
+// The Elastic Synthetics `request` parameter is similar to Playwright's
+// https://playwright.dev/docs/api/class-apirequestcontext[`APIRequestContext`], but
+// it comes built into Elastic Synthetics and doesn't have to be imported separately.
+// This reduces the code needed and allows you to make API requests in inline journeys.
```

Comment:
>We want to mention how this is different from the other request objects that is exposed by Playwright.

@vigneshshanmugam is this what you had in mind? Are there other differences?

Reply:
I was thinking something along the lines of

1. Top level `request` object exposed by synthetics have its own isolated cookie storage and its different from the  `context.request` and `page.request` as they shares cookie storage with the corresponding [BrowserContext](https://playwright.dev/docs/api/class-browsercontext).
2. If you want to control the creation of this request object - https://playwright.dev/docs/api/class-apirequest#api-request-new-context, you can do so by passing options via `--playwright-options` or `synthetics.config.ts` file.

Reply:
Thanks @vigneshshanmugam. How does this look? https://github.com/elastic/observability-docs/pull/2071/commits/0fc3fee31f57ace3777ca55b3c20ef339831e164

Here's what the whole section looks like:

<img width="883" alt="Screen Shot 2022-08-23 at 2 28 25 PM" src="https://user-images.githubusercontent.com/10479155/186249502-3fc51081-a179-4e5d-82d1-e7caab6f4d91.png">

Reply:
Looks great, Thanks @colleenmcginnis